build.gradle

```diff
@@ -11,7 +11,8 @@ repositories {
 
 dependencies {
   compile 'com.squareup:javapoet:1.7.0'
-  compile 'com.google.auto.value:auto-value:1.2'
+  compile 'com.google.auto.value:auto-value:1.6.3'
+  compile 'com.google.auto.value:auto-value-annotations:1.6.3'
```

Comment:
Is this artifact actually needed?

Reply:
To upgrade to a version after 1.6, we need to add that.([Check this](https://github.com/google/auto/commit/0c488d7f3a701d6947cd5e5cb28cf2c71d6b3424)) Otherwise I get errors "package com.google.auto.value does not exist"

Reply:
Sure but we don't need it in the processor. Are we using it for tests? It probably just needs added as a `testCompile` dep

Reply:
Yeah, the error comes from test. If I add it as testCompile, the build is successful.